client/protocols/backup.py

```diff
@@ -24,7 +24,7 @@ class BackupProtocol(models.Protocol):
     REVISION_ENDPOINT = '{prefix}://{server}/api/v3/revision/'
 
     def run(self, messages):
-        if self.args.local or self.args.export or self.args.restore:
+        if self.args.local or self.args.restore:
```

Comment:
Not in the scope of this PR - but I wonder if there's a better to way to specify when to not run a protocol. We repeat this logic in all protocols.

Maybe all protocols should have a method called `can_run(self)` that returns true or false before we attempt to run them.